docs/guide/start-prerequisites.md

```diff
@@ -0,0 +1,22 @@
+# What do you need to know
+
+Yii learning curve is not as steep as other PHP frameworks but still it requires some beforehand knowledge.
+
+## PHP
+
+Yii is a PHP framework so make sure you [read and understand language reference](http://php.net/manual/en/langref.php).
+
+## Object oriented programming
+
+Basic understanding of object oriented programming is required. If you're not familiar with it, check one of the many
+tutorials available such as [the one from tuts+](https://code.tutsplus.com/tutorials/object-oriented-php-for-beginners--net-12762).
+
+Note that the more complicated your application is the more advanced OOP concepts your should learn in order to successfully
+manage that complexity.
+
+## Command line and composer
+
+Yii extensively uses de-facto standard PHP package manager, [Composer](https://getcomposer.org/) so make sure you read
```

Comment:
Docs will be better if this composer cheat sheet link is added
http://composer.json.jolicode.com